docs/resources/compensating-research-participants.md

```diff
@@ -0,0 +1,62 @@
+0. Document **why** (benefits to objective, increased quality, reduced time in contractual terms) and **how** ([budget](https://ethn.io/incentives/calculator), number/range of participants, ethical considerations, timeline, method of compensation).
+0. Work with the project’s Client Engagement Manager on how to pass it by both the contracting officer and Truss Practice Lead or COO if it’s >$10k)
+0. Get practice lead/COO approval
+0. Get contracting officer’s approval
+
+### For time & materials (T&M) contracts
+Ideally, take it entirely out of Truss’ hands by writing into the contract—either from the beginning or as a mod ([🔒 DOL’s](https://drive.google.com/file/d/1Un_A0sPXXN-UXCHaoGTLe9lpfbe67MI7/view) for example)—that the client(s) will be responsible and accountable for compensating research participants. Additionally, write a decision record with the client’s project team ([🔒 DOL’s](https://docs.google.com/document/d/1X1GOyJIhsw9Zs2Qu_CZ7RPHOljUP3HRpFOncALei6c0/edit#heading=h.pjv8vxtpra0j) for example) to clarify who and how someone from the client should compensate participants. For example:
+
+- All compensation will be paid out to participants from [sub-agency]
+- Research participants will be paid $[D] per hour, usually in a [M]-minute session
+- Targeting [S] sessions per month for [T] months with a budget of $[B]
+- [Person] will use the recruiting spreadsheet, which has participants’ name, email, when/whether they attended, how much to pay them, and a running burndown against the total budget
+- [Person] will purchase the gift cards and email it to participants by the end of the week of their session.
+
+## Examples
+
+### DOL
+Our partner USDS managed recruiting and most of the research ops, including compensation. We did not have [Paperwork Reduction Act](https://pra.digital.gov/) (PRA) approval to screen participants via a survey, and not much time to recruit (started at the same time as other research work), so USDS relied on advocacy agencies to recruit for us, allowing us to reach a fairly diverse group of folks. USDS’s researcher then used their budget to buy Visa gift cards at $50 per person, emailed to participants after all sessions wrapped (a few weeks long total). They did this for various folks in the US, but not for the federal and state employees we interviewed for other research (see [“government employees”](#government-employees) above). 
+
+Note that it took USDS, their (White House) lawyers, and procurement officers two years to get approval for the concept of compensating research participants. The researcher got that theoretical clearance but needed to provide a user justification memo for this particular research series. Initially they just tried a paragraph, but that reportedly wasn’t enough info. So they tried again with a page and a half of detail of why they should pay folks, and that did work but with no feedback about why. Their details included:
```

Comment:
We don't have a copy of that page and a half, do we? :) If so, including it here could be helpful!

Reply:
we do not because it was USDS, the idea here was that the volume seemed to matter. I added a little blurb to that effect